hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/job/TensorFlowJob.groovy

```diff
@@ -118,10 +46,7 @@ class TensorFlowJob extends HadoopJavaProcessJob {
    * @param amMemory ApplicationMaster memory, in MB, for the TensorFlow job
    */
   @HadoopDslMethod
-  void amMemory(int amMemory) {
-    this.amMemory = amMemory;
-    setJobProperty("am_memory", this.amMemory);
-  }
+  void amMemory(int amMemory);
```

Comment:
a comment from design doc, we should add unit here - amMemoryMb, or we need to support `m` and `g`